core/templates/dev/head/dashboard/MyExplorations.js

```diff
@@ -37,11 +37,11 @@ oppia.controller('MyExplorations', [
     };
 
     $rootScope.loadingMessage = 'Loading';
-    $http.get('/myexplorationshandler/data').success(function(data) {
-      $scope.explorationsList = data.explorations_list;
+    $http.get('/myexplorationshandler/data').then(function(response) {
+      $scope.explorationsList = response.data.explorations_list;
```

Comment:
Will do, I have tried to use the "var data = response.data;" convention whenever response.data is used more than 2 times.